src/OrchardCore.Modules/OrchardCore.Facebook/Manifest.cs

```diff
@@ -39,3 +40,10 @@
         "OrchardCore.Recipes.Core",
     }
 )]
+
+[assembly: Feature(
+    Id = FacebookConstants.Features.Pixel,
+    Name = "Facebook Pixel",
```

Comment:
Add a note about this to the module's docs.

Reply:
done